blog-service/2024-01-15-ui.md

```diff
@@ -0,0 +1,19 @@
+---
+title: New and Improved Sumo Logic UI
+image: https://www.sumologic.com/img/logo.svg
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Update - We've rolled out our new and improved Sumo Logic user interface (UI) with a simplified navigation. Here are some ways we’ve streamlined the overall experience:
+
+* You can access all Observability and Security features (like SIEM and SOAR) in one unified navigation. essential to operate and secure modern digital apps.
+* To streamline navigation of the above features, we've moved Administration, Configuration, Help, and User Preferences, to the global application toolbar located on the top right side navigation.
+* We've eliminated in-app tabs for a more straightforward and responsive navigation, faster performance, better usability, and consistent browsing experience similar to other web applications. You can now utilize browser tabs, session history, the back button, favorites, and other native browser capabilities you're familiar with.
+* URLs are now stateful, making it easier for you to share the most up-to-date content with your teammates. Any changes you make in your UI will be reflected globally in the URL parameters.
+
+We'd love your feedback! Just click the **Feedback** button at the bottom of the left-side nav or reach out to our [Sumo Dojo Slack community](https://sumodojo.slack.com/) #sumo-new-ui channel.
+
+For further assistance, contact [Support](https://support.sumologic.com/support/s/).
```

Comment:
Shouldn't the release note link to the following article?
https://help.sumologic.com/docs/get-started/sumo-logic-ui/

Reply:
oh my goodness, yikes!! good catch.